doc/source/timeseries.rst

```diff
@@ -265,6 +265,22 @@ Typical epoch stored units
    pd.to_datetime([1349720105100, 1349720105200, 1349720105300,
                    1349720105400, 1349720105500 ], unit='ms')
 
+Conversion of float epoch times can lead to inaccurate and unexpected results.
```

Comment:
you mean the line as note or the whole section?

also unit is taken as the base but precision is taken from float. First example is exactly the problem I've had at the beginning.

```
In [36]: pd.to_datetime(1490195805.433502912, unit='s')
Out[36]: Timestamp('2017-03-22 15:16:45.433503')
```
It neither `s` nor `ns` in the example.